js/ui/scheduler/ui.scheduler.subscribes.js

```diff
@@ -759,7 +772,7 @@ const subscribes = {
         return SchedulerTimezones.getTimezonesIdsByDisplayName(displayName);
     },
 
-    getTargetedAppointmentData: function(appointmentData, appointmentElement) {
+    getTargetedAppointmentData: function(appointmentData, appointmentElement, convertDates) {
```

Comment:
The `convertDates` is not necessary

Reply:
Yes, indeed, the variable is no longer needed.